### ISSUE TITLE
BN View?

### ISSUE THREAD

**Issue by erichmccartney** [mattgwagner/Battery-Commander#790]
I'm sure this is a big ask especially since even I can see how the architecture of this app is a bit taxing to update.

But perhaps my open contribs bought some coding alimony 😃

Perhaps there is something we could hack together for just the evaluation screen since I see that as the big money maker in the app that will really drive adoption.

- improve the clarity of which unit is in view
- add quick access filters for subordinate units
- add unit column when viewing bn level view

![image](https://user-images.githubusercontent.com/29133992/126911338-d00c65b8-b4ed-450b-bfe6-38e9da5d34b4.png)

**Comment by mattgwagner** [comment]
I don't think this would be a ton of work -- the biggest change would be the data model evolving to support a higher headquarters relationship. It's probably something that would come into play on several screens, and probably should be supported by a distinct selection dropdown in the header for unit context.

**Comment by erichmccartney** [comment]
The addition of unit column is great via #795 this sort of accomplishes the BN view for OERs at least since the BC being on all OERs made the cross-pollination happen.

The unit filter buttons would still help NCOERs

However, now I'm seeing that an OER / NCOER toggle could be useful since the conversation is typically split like that.

Warrants are grouped into the OER category
